docs/community_packages.md

```diff
@@ -4,13 +4,17 @@
 
 Below is a list of packages contributed by the community. These packages normally follow the same installation procedure, however, its best you consult the documentation of the package itself in case there is anything special you need to get it working.
 
+Applying community packages to SeAT requires modifying your .env file located at /var/www/seat/.env for bare metal installs and /opt/seat-docker/.env for docker installs. To add the package(s) you must uncomment SEAT_PLUGINS by removing the '#' and entering the package(s) to be installed following the guidelines provided.
```

Comment:
Bare metal installs don't have the same `.env` file as docker installs. They have to manually do the composer commands that the docker install will do for you.

Reply:
I've updated the document removing the bare metal references I added and cleaned up the Docker instructions per my last commit.